app/templates/index.hbs

```diff
@@ -2,6 +2,19 @@
   <h1>Welcome to my site!</h1>
 </div>
 
+{{#if this.isDev}}
```

Comment:
Can you please verify if this check is working or not? I am trying to visit the same with `dev=true` flag but it isn't working.
![image](https://user-images.githubusercontent.com/78433013/227739707-5f36babb-9f6f-425e-8d98-01d8e04c6c20.png)

Reply:
resolved it in last commit